src/AvTranscoder/properties/VideoProperties.cpp

```diff
@@ -338,11 +338,11 @@ size_t VideoProperties::getBitRate() const
     // return bit rate of stream if present or VBR mode
     if(_codecContext->bit_rate || _codecContext->rc_max_rate)
         return _codecContext->bit_rate;
+    LOG_WARN("The bitrate of the stream '" << _streamIndex << "' of file '" << _formatContext->filename << "' is unknown.")
 
     if(_levelAnalysis == eAnalyseLevelHeader)
     {
-        LOG_WARN("The bitrate of the stream '" << _streamIndex << "' of file '" << _formatContext->filename << "' is unknown. "
-                "Need a deeper analysis: see eAnalyseLevelFirstGop.")
+        LOG_INFO("Need a deeper analysis: see eAnalyseLevelFirstGop.")
```

Comment:
I think the info level is fine here. And the warning part of the message is always print as a warning few lines above. Don't you think?